_benchmark/user-guide/target-throughput.md

```diff
@@ -16,13 +16,15 @@ OpenSearch Benchmark has two testing modes, both of which are related to through
 
 ## Benchmarking mode
 
-When you do not specify a `target-throughput`, OpenSearch Benchmark latency tests are performed in *benchmarking mode*. In this mode, the OpenSearch client sends requests to the OpenSearch cluster as fast as possible. After the cluster receives a response from the previous request, OpenSearch Benchmark immediately sends the next request to the OpenSearch client. In this testing mode, latency is identical to service time.
+When `target-throughput` is set to `0`, OpenSearch Benchmark latency tests are performed in *benchmarking mode*. In this mode, the OpenSearch client sends requests to the OpenSearch cluster as fast as possible. After the cluster receives a response from the previous request, OpenSearch Benchmark immediately sends the next request to the OpenSearch client. In this testing mode, latency is identical to service time.
+
+OpenSearch Benchmark issues one request at a time for a single-client thread, which is specified as `search-clients` in the workload parameters. 
```

Comment:
What does "single-client" thread mean? And how does the user specify it in `search-clients`?

Reply:
My understanding is that "single-client thread" means a single connection per request, hence one request per thread. The `search-clients` setting specifies the number of single-threaded clients the workload uses.  

Should we clarify this @IanHoang? Is my understanding correct at @IanHoang?